code/style.css

```diff
@@ -2,6 +2,236 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+/* .active + .answer {
+  /* some styling here 
+ */
+
+ * {
+  margin: 0;
+}
+
+body {
+  background: #fff;
+  margin:0; 
+}
+
+/**** HEADER ****/
+
+header {
+background-color: #fff;
+text-align: center;
+
+}
+
+header img {
+  height: 100px;
+  padding: 10px;  
+}
+
+.hero-image {
+  height: 380px;
+  background-image: url('balloons.jpg');
+  background-repeat: no-repeat;
+  background-size:cover;
+  position: relative;
+}
+
+/* .hero-image-text {
+  font-size: 20px;
+  text-align: center;
+} */
+ 
+
+.hero-image-text {
+  position: absolute;
+  text-align: center;
+  top: 42%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-family: 'Lobster', cursive;
+  Font-Size: 30px;
+  color:  #48463c;
+  background: rgb(225,225,225, 0.7);
+  border: 1px solid  #0b6ba8;
+  padding: 20px;
+}  
+
+
+h2 {
+  font-family: 'Lobster', cursive;
+  Font-Size: 32px;
+  margin: 10px;
+  color:#48463c;
+}
+
+label {
+  font-family: 'Roboto Condensed', sans-serif;
+  font-size: 16px;
+}
+/**** FLEX PARENT ****/
+
+.flex-parent {
+  display: flex;
+  height: 800px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  margin: 15px;
+  
+}
+
+.flex-child {
+  width: 345px;
+  border: 5px solid #0b6ba8;
+  border-radius: 5px;
+  
+}
+
+/**** ACCORDION ****/
+
+.accordion {
+  background: lightblue;
+  
+}
+
+.accordion-element {
+  border-top: 1px solid #0b6ba8;
+  padding:0px
+}
+
+
+.accordion-header {
+  background: #fff;
+  font-family: 'Roboto Condensed', sans-serif;
+  Font-Size: 22px;
+  margin: 0;
+  padding: 20px;
```

Comment:
To make it more obvious that something happens if you'd click on this element, you could try to change to cursor. It could maybe change on hover? Of course it's not necessary now but it might be nice to know about, here's some reading:
https://developer.mozilla.org/en-US/docs/Web/CSS/cursor